.github/scripts/actionUtils.js

```diff
@@ -0,0 +1,23 @@
+// tagSummaryUtils.js
```

Comment:
If this is supposed to be the file's name, it is not, Please adjust either this line to match the filename or update the filename and anywhere else it is called.